fvtest/porttest/si.cpp

```diff
@@ -500,6 +501,7 @@ TEST(PortSysinfoTest, sysinfo_test_sysinfo_env_iterator)
 	J9SysinfoEnvElement element;
 	void *buffer = NULL;
 	uint32_t bufferSizeBytes = 0;
+	int l = 0;
```

Comment:
This variable doesn't appear to be needed (and the name is a particularly bad choice `l` looks the same as `1` in some fonts).

Reply:
Yes I see it was removed via https://github.com/eclipse/omr/commit/c299895763778a5baa252278a298d9501e04f252 , I think another rebase may be needed, otherwise ill just get rid of it if that doesn't work